### PR TITLE
Deprecate powershell v6

### PR DESCRIPTION
**Pull request by gzuber**
According to https://github.com/Azure/azure-cli/issues/14649, powershell v6 is approaching end of life and should be deprecated.

### CODE REVIEW

**Comment by pragnagopa** [comment]
stack should not be removed. Instead set isDeprecated to true.

**Comment by pragnagopa** [comment]
@Francisco-Gamino - Please use https://github.com/Azure/azure-functions-ux/pull/5201/files as an example and send a PR updating to official stacks

cc @ammanifold as FYI

**Comment by Francisco-Gamino** [comment]
@gzuber : Do you still need to make a code change? Or setting `isDeprecated` to `true` will block the creation of the PowerShell 6.2 function app in the Az CLI?

**Review by pragnagopa — CHANGES_REQUESTED** [review]
Please update the PR

**Comment by ammanifold** [comment]
I thought we didn't want to update this until we have official GA for PS7

**Comment by gzuber** [comment]
> @gzuber : Do you still need to make a code change? Or setting `isDeprecated` to `true` will block the creation of the PowerShell 6.2 function app in the Az CLI?

@Francisco-Gamino yeah this will still require a code change. Even if the CLI had support for isDeprecated, I'd still have to update the stacks API file (right now we just include the json in the CLI build). But I have support for isHidden and isPreview so it shouldn't be difficult to add.

**Comment by pragnagopa** [comment]
Thanks @gzuber - changes LGTM

@Francisco-Gamino  - can you ACK on this change and answer concern raised by @ammanifold 

> I thought we didn't want to update this until we have official GA for PS7 ?

**Comment by Francisco-Gamino** [comment]
> I thought we didn't want to update this until we have official GA for PS7

@stefanushinardi could you please help answer @ammanifold question?

**Comment by stefanushinardi** [comment]
@ammanifold is right, we should wait to release this until PS7 GA announcement. If this commit gets merged today, when would be it be released?

**Comment by pragnagopa** [comment]
@stefanushinardi thanks for the input

>  If this commit gets merged today, when would be it be released?

To avoid coordinating multiple release schedules across multiple repos, let's wait on updating stacks. We should update the stacks after PS7 GA annoucement. Do you have an ETA?

**Comment by stefanushinardi** [comment]
@pragnagopa  ETA that we are currently tracking is 3rd week of August

**Comment by pragnagopa** [comment]
@gzuber / @Francisco-Gamino  - Since the changes required in the tooling are small, lets hold off on merging these changes.

Please add support for `isDeprecated` flag. Once this change is merged, only work remaining would be to update stacks in tooling repos.

**Comment by Francisco-Gamino** [comment]
> @gzuber / @Francisco-Gamino - Since the changes required in the tooling are small, lets hold off on merging these changes.
 Please add support for isDeprecated flag. Once this change is merged, only work remaining would be to update stacks in tooling repos.

Sounds good. Thanks @pragnagopa .

**Comment by stefanushinardi** [comment]
@pragnagopa @gzuber @Francisco-Gamino, the PS7 GA announcement of  is now live: https://azure.microsoft.com/en-us/updates/azure-functions-powershell-7-support-is-now-generally-available/. Please merge this with the next release. Thanks!

**Review by pragnagopa — APPROVED** [review]

**Comment by pragnagopa** [comment]
@Francisco-Gamino - please send a PR updating stacks in Ux repo.

**Comment by Francisco-Gamino** [comment]
> @Francisco-Gamino - please send a PR updating stacks in Ux repo.

@pragnagopa  - will do and tag you in the PR.

**Comment by Francisco-Gamino** [comment]
Here is the link to the UX PR: https://github.com/Azure/azure-functions-ux/pull/5230

**Pull request merged**
